packages/appservices-patternfly-crossconsole/.cache_1c493o9

```diff
@@ -0,0 +1 @@
gCount":0},"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/ContextSelectorMenuList.tsx",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/ContextSelectorToggle.tsx",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/CrossNavContextSelector.tsx",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/CrossNavContextSelectorConstants.tsx",["47"],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/CrossNavContextSelectorItem.tsx",["48","49"],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavContextSelector/index.ts",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavHeader/CrossNavHeader.tsx",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/CrossNavHeader/index.ts",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/LoadingPage/LoadingPage.tsx",["50","51"],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/components/LoadingPage/index.ts",[],"/Users/dlabaj/repos/appservices-patternfly/packages/appservices-patternfly-crossconsole/src/index.ts",[],{"ruleId":null,"message":"52","line":5,"column":3,"endLine":5,"endColumn":64,"severity":1,"nodeType":null},{"ruleId":null,"message":"53","line":42,"column":5,"endLine":42,"endColumn":35,"severity":1,"nodeType":null},{"ruleId":null,"message":"52","line":47,"column":5,"endLine":47,"endColumn":66,"severity":1,"nodeType":null},{"ruleId":"54","severity":2,"message":"55","line":13,"column":17,"nodeType":"56","endLine":13,"endColumn":75,"fix":"57"},{"ruleId":"54","severity":2,"message":"58","line":14,"column":21,"nodeType":"56","endLine":14,"endColumn":137,"fix":"59"},"'//eslint-disable-next-line' has no effect because you have 'noInlineConfig' setting in your config (BaseConfig).","'/*eslint-disable-next-line*/' has no effect because you have 'noInlineConfig' setting in your config (BaseConfig).","pf-codemods/refactor-breakpointmods","Removed breakpointMods prop from Flex in favor of spacer, spaceItems, grow, shrink, flex, direction, alignItems, alignContent, alignSelf, align, justifyContent, display, fullWidth and flexWrap","JSXOpeningElement",{"range":"60","text":"61"},"Removed breakpointMods prop from FlexItem in favor of spacer, grow, shrink, flex, alignSelf, align, and fullWidth",{"range":"62","text":"63"},[421,472],"direction={{\"default\":\"column\"}} ",[504,609],"spacer={{\"default\":\"spacerXl\"}} alignSelf={{\"default\":\"alignSelfCenter\"}} "]
```

Comment:
looks like a temp cache file, probably not intended to merge in with your other changes